pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -8,7 +8,10 @@
 
 package changefeedbase
 
-import "github.com/cockroachdb/cockroach/pkg/sql"
+import (
+	"github.com/cockroachdb/cockroach/pkg/ccl/importccl"
```

Comment:
Let's not add this dependency.  Code reuse is good an all, but we're probalby importing a small utility function -- we can reimplement it ourselves.

Reply:
duplicated in `options.go`